sandbox/src/timebar/timebar.js

```diff
@@ -1,75 +1,91 @@
 import dayjs from 'dayjs'
+
+// mocks
 import activityMock from './mocks/activity'
+import trackMock from './mocks/trackWithFishingSpeedAndCourse'
 import eventsMock from './mocks/events'
 import vesselEventsMock from './mocks/vesselEvents'
 // import groupVesselEvents from './mocks/groupVesselEvents'
 
+// timebar
 import Timebar, {
   TimebarActivity,
   TimebarEvents,
   TimebarVesselEvents,
+  TimebarTracks,
+  TimebarHighlighter,
+  getHumanizedDates,
 } from '@globalfishingwatch/map-components/src/timebar'
+import { geoJSONTrackToTimebarTrack, geoJSONTrackToTimebarFeatureSegments } from '@globalfishingwatch/map-components/src/timebar/utils'
 
 import './timebar.css'
 
-const HOVER_DELTA = 10
-
-// --- TODO This should be inside Activity.jsx - let's have charts deal with data formats
-const maxActivity = activityMock.reduce((acc, current) => Math.max(acc, current.value), 0)
-// const activity = {};
-// activityMock.forEach((d) => { activity[dayjs(d.date).format('YYYY-MM-DD')] = d.value / maxActivity; });
-const activity = activityMock.map((d) => ({
-  date: d.date,
-  id: dayjs(d.date).format('YYYY-MM-DD'),
-  value: d.value / maxActivity,
-}))
-// ---
-
-const getTime = (dateISO) => new Date(dateISO).getTime()
-const getDeltaMs = (start, end) => getTime(end) - getTime(start)
-const getDeltaDays = (start, end) => getDeltaMs(start, end) / 1000 / 60 / 60 / 24
-const isMoreThanADay = (start, end) => getDeltaDays(start, end) >= 1
-const getHumanizedDates = (start, end) => {
-  const format = isMoreThanADay(start, end) ? 'MMM D YYYY' : 'MMM D YYYY HH[h]'
-  const humanizedStart = dayjs(start).format(format)
-  const humanizedEnd = dayjs(end).format(format)
-  return { humanizedStart, humanizedEnd }
-}
 
-// const groupedVesselEvents = groupVesselEvents(vesselEventsMock)
-// console.log(groupedVesselEvents)
-// console.log(groupedVesselEvents.map(e => e.type))
+const HOVER_DELTA = 8
+
 
-const initialStart = '2018-04-01T00:00:00.000Z'
-const initialEnd = '2019-03-31T00:00:00.000Z'
+const trackActivityMock = geoJSONTrackToTimebarFeatureSegments(trackMock)
+
+const getTrackActivityMockForSubChart = memoize((activity, currentSubChart) =>
+  activity.map(segment =>
+    segment.map(item => ({
+      ...item,
+      value: item[currentSubChart]
+    })
+  ))
+)
 
-const absoluteStart = '2017-01-01T00:00:00.000Z'
-const absoluteEnd = initialEnd
+const getTrackMockForSubChart = memoize((trackMock) => {
+  const timebarTrack = geoJSONTrackToTimebarTrack(trackMock)
+  return [
+    timebarTrack,
+    { ...timebarTrack, color: '#00ff00' },
+    { ...timebarTrack, color: '#00ffff' },
+    { ...timebarTrack, color: '#ffff00' }
+  ]
+})
+
+
+// const initialStart = new Date(trackActivityMock[0][0].date).toISOString()
+const initialStart = '2012-01-01T00:00:00.000Z'
+const initialEnd = new Date(trackActivityMock[trackActivityMock.length-1][0].date).toISOString()
+const absoluteStart = '2012-01-01T00:00:00.000Z'
+const absoluteEnd = '2019-08-31T00:00:00.000Z'
```

Comment:
@nerik what's this hardcoded date for?

Reply:
This is just a sandbox place to test the components, in this case is a page which contains the timebar and use this values to see the timebar working with mocks:
![image](https://user-images.githubusercontent.com/10500650/64319831-8fcf3200-cfbd-11e9-9882-9a23f6722242.png)